### PR TITLE
Remove commercecode as parameter

### PR DESCRIPTION
**Pull request by iescaida**
Remove commerce code as parameter since is coming in the header, also swapped type for maxAmount from double to String to allow optionality

### CODE REVIEW

File: src/main/java/cl/transbank/webpay/example/controller/patpass/comercio/PatpassComercioController.java [hunk 1]
```diff
@@ -33,23 +36,40 @@ public PatpassComercioController() {
...
         String mobileNumber = "123456723";
         String patpassName = "nombre del patpass";
-        String personEmail = "otromail@persona.cl";
-        String commerceEmail = "otrocomercio@comercio.cl";
+        String personEmail = "alba.cardenas@continuum.cl";
+        String commerceEmail = "alba.cardenas@continuum.cl";
         String address = "huerfanos 101";
         String city = "Santiago";
 
+        /*String url = "https://www.help.cl/contrata-online/patpassWs/response.php";
```
Comment (a1b4) [inline src/main/java/cl/transbank/webpay/example/controller/patpass/comercio/PatpassComercioController.java]: can you remove this commented code?

**Review by a1b4 — CHANGES_REQUESTED** [review]

**Comment by goncafa** [comment]
@iescaida Why did you have to change the maxAmount param to String instead a Double? If the problem that param should be optional you could manage it with the Double wrapper.

File: src/main/java/cl/transbank/webpay/example/controller/patpass/comercio/PatpassComercioController.java (hunk 1 above)
Comment (iescaida) [reply src/main/java/cl/transbank/webpay/example/controller/patpass/comercio/PatpassComercioController.java]: done

**Comment by iescaida** [comment]
> @iescaida Why did you have to change the maxAmount param to String instead a Double? If the problem that param should be optional you could manage it with the Double wrapper.

@goncafa 
I was avoiding the hassle of using a wrapper, since they can pass an empty maxAmount and as far as I know it can't be an empty Double right

**Comment by goncafa** [comment]
> > @iescaida Why did you have to change the maxAmount param to String instead a Double? If the problem that param should be optional you could manage it with the Double wrapper.
> 
> @goncafa
> I was avoiding the hassle of using a wrapper, since they can pass an empty maxAmount and as far as I know it can't be an empty Double right

@iescaida But you can use null instead and you could overload a method in order to support receive the param or not.

**Review by goncafa — APPROVED** [review]

**Review by a1b4 — APPROVED** [review]
nice work 🎉

**Pull request merged**